addons/mail/static/src/messaging/component/notification_popover/notification_popover.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.NotificationPopover" owl="1">
+        <div class="o_NotificationPopover">
+            <t t-foreach="notifications" t-as="notification" t-key="notification.localId">
+                <div class="o_NotificationPopover_notification">
+                    <t t-if="notification.notification_status === 'sent'">
+                        <i class='o_NotificationPopover_icon fa fa-check' title="Sent" role="img"/>
+                    </t>
+                    <t t-if="notification.notification_status === 'bounce'">
+                        <i class='o_NotificationPopover_icon fa fa-exclamation text-danger' title="Bounced" role="img"/>
+                    </t>
+                    <t t-if="notification.notification_status === 'exception'">
+                        <i class='o_NotificationPopover_icon fa fa-exclamation text-danger' title="Error" role="img"/>
+                    </t>
+                    <t t-if="notification.notification_status === 'ready'">
+                        <i class='o_NotificationPopover_icon fa fa-send-o' title="Ready" role="img"/>
+                    </t>
+                    <t t-if="notification.notification_status === 'canceled'">
+                        <i class='o_NotificationPopover_icon fa fa-trash-o' title="Canceled" role="img"/>
+                    </t>
+                    <span class="o_NotificationPopover_partnerName" t-esc="notification.partner.nameOrDisplayName"/>
```

Comment:
Due to template repetition, wouldn't we want to make "class" and "title" getters in the JS and define the `<i>` only once?

Reply:
agree, makes more sense a single `<i>` with getters.

Reply:
done